archinstall/lib/menu/global_menu.py

```diff
@@ -87,82 +87,82 @@ def _setup_selection_menu_options(self):
 				dependencies=['harddrives'])
 		self._menu_options['swap'] = \
 			Selector(
-				_('Use swap'),
+				_('Swap'),
 				lambda preset: ask_for_swap(preset),
 				default=True)
 		self._menu_options['bootloader'] = \
 			Selector(
-				_('Select bootloader'),
+				_('Bootloader'),
 				lambda preset: ask_for_bootloader(storage['arguments'].get('advanced', False),preset),
 				default="systemd-bootctl" if has_uefi() else "grub-install")
 		self._menu_options['hostname'] = \
 			Selector(
-				_('Specify hostname'),
+				_('Hostname'),
 				ask_hostname,
 				default='archlinux')
 		# root password won't have preset value
 		self._menu_options['!root-password'] = \
 			Selector(
-				_('Set root password'),
+				_('root password'),
 				lambda preset:self._set_root_password(),
 				display_func=lambda x: secret(x) if x else 'None')
 		self._menu_options['!superusers'] = \
 			Selector(
-				_('Specify superuser account'),
+				_('Superuser account'),
 				lambda preset: self._create_superuser_account(),
 				default={},
 				exec_func=lambda n,v:self._users_resynch(),
 				dependencies_not=['!root-password'],
 				display_func=lambda x: self._display_superusers())
 		self._menu_options['!users'] = \
 			Selector(
-				_('Specify user account'),
+				_('User account'),
 				lambda x: self._create_user_account(),
 				default={},
 				exec_func=lambda n,v:self._users_resynch(),
 				display_func=lambda x: list(x.keys()) if x else '[]')
 		self._menu_options['profile'] = \
 			Selector(
-				_('Specify profile'),
+				_('Profile'),
 				lambda preset: self._select_profile(preset),
 				display_func=lambda x: x if x else 'None')
 		self._menu_options['audio'] = \
 			Selector(
-				_('Select audio'),
+				_('Audio'),
 				lambda preset: ask_for_audio_selection(is_desktop_profile(storage['arguments'].get('profile', None)),preset),
 				display_func=lambda x: x if x else 'None',
 				default=None
 			)
 		self._menu_options['kernels'] = \
 			Selector(
-				_('Select kernels'),
+				_('Kernels'),
 				lambda preset: select_kernel(preset),
 				default=['linux'])
 		self._menu_options['packages'] = \
 			Selector(
-				_('Additional packages to install'),
+				_('Additional packages'),
 				# lambda x: ask_additional_packages_to_install(storage['arguments'].get('packages', None)),
 				ask_additional_packages_to_install,
 				default=[])
 		self._menu_options['additional-repositories'] = \
 			Selector(
-				_('Additional repositories to enable'),
+				_('Additional repositories'),
```

Comment:
I would probably cange this to "Optional repositories" instead, while we're at it.